.travis.yml

```diff
@@ -2,5 +2,5 @@ language: python
 python:
   - 2.7
 install:
-  - pip install .
+  - pip install . --pre
```

Comment:
for a release, should we remove the `--pre` option? if yes, we have to document it

Reply:
For the release we are specifying the exact version in setup.py (for example ==3.8.1) so the --pre will not affect anything and install the exact same package version